master/buildbot/status/web/jsontestresults.py

```diff
@@ -23,9 +24,17 @@ def __init__(self, log, step_status):
         self.log = log
         self.step_status = step_status
 
+    def get_artifacts_path(self, build):
+        artifacts = build.artifacts.values() if build.artifacts else []
```

Comment:
@kkateq the global configuration is not accessible here, and the artifact path is pass as a parameter in the upload download steps. let's check this together.